app/controllers/reviewables_controller.rb

```diff
@@ -318,4 +325,8 @@ def meta_types
   def ensure_can_see
     Guardian.new(current_user).ensure_can_see_review_queue!
   end
+
+  def with_deleted_content
+    Post.unscoped { Topic.unscoped { PostAction.unscoped { yield } } }
```

Comment:
Maybe for these we should try use the Trashable's `with_deleted`, since I think unscoped is a bit more of a hammer?

Reply:
we can't do that (at least I have not figured out how to) ... there are polymorphic associations that need to conditionally include trashable items so we would need carry context within the relations we generate.